src/molecules/formfields/TextField/index.js

```diff
@@ -13,16 +13,55 @@ import { renderTooltip } from 'utils/fieldUtils';
 import styles from './text_field.module.scss';
 
 class TextField extends React.Component {
-  render() {
+  getClasses = () => {
```

Comment:
this seems like a pattern, want to know the rule for all of these changes

Reply:
`complexity` - I had to refactor some of the code out of the `render` function because it was too complex

Reply:
I don't think it's a huge deal to keep this as-is but it might be better to just remove that rule since I don't really like linters determining our code complexity for us

Reply:
file revered back to master and complexity ignored